src/control-plane/backend/config/dictionary.json

```diff
@@ -86,78 +86,120 @@
         {
           "name": "_first_open",
           "description": {
-            "en-US": "The first time a user opens an app",
-            "zh-CN": "用户首次打开应用"
+            "en-US": "When the user launches an app the first time after installation",
+            "zh-CN": "当用户安装后第一次启动应用程序时"
           }
         },
         {
           "name": "_session_start",
           "description": {
-            "en-US": "The start of a session",
-            "zh-CN": "会话开始"
+            "en-US": "When a user first open the app or a user returns to the app after 30 minutes of inactivity period",
+            "zh-CN": "当用户首次打开App或用户在 30 分钟不活动后返回App时"
           }
         },
         {
           "name": "_screen_view",
           "description": {
-            "en-US": "The user views a screen",
-            "zh-CN": "用户浏览屏幕"
+            "en-US": "When new screen is opens",
+            "zh-CN": "当新的屏幕打开时"
           }
         },
         {
           "name": "_user_engagement",
           "description": {
-            "en-US": "The user interacts with the app",
-            "zh-CN": "用户与应用交互"
+            "en-US": "When user navigates away from current screen and the screen is in focus for at least one second",
+            "zh-CN": "当用户离开当前屏幕并且屏幕处于焦点状态至少一秒钟时"
           }
         },
         {
           "name": "_app_start",
           "description": {
-            "en-US": "The app is launched",
-            "zh-CN": "应用启动"
+            "en-US": "Every time the app goes to visible",
+            "zh-CN": "每次App变为可见时"
           }
         },
         {
           "name": "_app_end",
           "description": {
             "en-US": "The app is closed",
```

Comment:
It looks inconsistent with Chinese. Should be `the app is invisiable` or hidden? @zhu-xiaowei